Documentation/ceph-upgrade.md

```diff
@@ -56,6 +56,16 @@ released, the process of updating from v1.2.0 is as simple as running the follow
 kubectl -n rook-ceph set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.2.1
 ```
 
+With support of ceph-csi v2.0.0 ClusterRoles required for csi-attacher and
```

Comment:
How about the following paragraph?

"If you want to enable the CSI v2.0 driver, additional steps are needed in the latest Rook patch release. There are some new features in 2.0 such as supporting resizing volumes and support for a new csi attacher. If you don't need these features now, you can skip this step for the patch upgrade. "

Then I'd suggest a new section such as this:

## Enabling the CSI v2.0 driver

To enable the CSI v2.0 driver, you will need to apply updated RBAC settings:
```sh
kubectl apply -f upgrade-from-v1.2-apply.yaml
```

Now specify the CSI v2.0 images to run the new version:
```
<insert instructions for updating the images>
```